### DISCUSSION TITLE
failed: WebSocket is closed before the connection is established.

### DISCUSSION THREAD

**Discussion by fauzifadillah** [beyondcode/laravel-websockets#805] [Q&A]
My application is deployed on Shared Hosting with SSH Access and Port 6001 opened. But when I try to establish the connection the console says:

failed: WebSocket is closed before the connection is established.

here's my config:
`
// broadcasting.php
'connections' => [

        'pusher' => [
            'driver' => 'pusher',
            'key' => env('PUSHER_APP_KEY'),
            'secret' => env('PUSHER_APP_SECRET'),
            'app_id' => env('PUSHER_APP_ID'),
            'options' => [
                'cluster' => env('PUSHER_APP_CLUSTER'),
                // 'useTLS' => true,
                'encrypted' => false,
                'host' => 'back-office.itsloco.co.id',
                'port' => 6001,
                'scheme' => 'http',
            ],
        ],`

please help:(

**Comment by Ghustavh97** [comment]
@fauzifadillah for me is was a the `options.host` option that was the problem. I changed it from the `website url` to the `server ip address` eg.

```diff
- 'host' => 'back-office.itsloco.co.id',
+ 'host' => '127.0.0.1',
```

and then run `php artisan cache:clear`

**Comment by ShaikhDanish04** [comment]
I tried your solution @Ghustavh97 but it is not working
